checkov/sca_package/output.py

```diff
@@ -44,18 +39,34 @@ def output_row(self) -> List[str]:
 
 
 def create_report_record(
-    rootless_file_path: str, file_abs_path: str, check_class: str, vulnerability_details: Dict[str, Any]
+    rootless_file_path: str, file_abs_path: str, check_class: str, vulnerability_details: Dict[str, Any],
+    runner_filter: RunnerFilter = RunnerFilter()
 ) -> Record:
     package_name = vulnerability_details["packageName"]
     package_version = vulnerability_details["packageVersion"]
     cve_id = vulnerability_details["id"].upper()
     severity = vulnerability_details.get("severity", DEFAULT_SEVERITY)
+    # sanitize severity names
+    if severity == "moderate":
+        severity = "medium"
     description = vulnerability_details.get("description")
     resource = f"{rootless_file_path}.{package_name}"
 
     check_result: _CheckResult = {
         "result": CheckResult.FAILED,
     }
+
+    if runner_filter.skip_cve_package and package_name in runner_filter.skip_cve_package:
+        check_result = {
+            "result": CheckResult.SKIPPED,
+            "suppress_comment": f"Filtered by package '{package_name}'"
+        }
+    elif SEVERITY_RANKING[severity] > SEVERITY_RANKING[runner_filter.min_cve_severity]:
```

Comment:
nice job leveraging my `SEVERITY_RANKING` map, which I actually added for sorting 😄